OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -442,4 +443,15 @@ public interface ITargetableCells
 
 	[RequireExplicitImplementation]
 	public interface IPreventsShroudReset { bool PreventShroudReset(Actor self); }
+
+	[RequireExplicitImplementation]
+	public interface IBotModuleInfo : ITraitInfo { }
```

Comment:
We have moved away from making interfaces inherit `ITraitInfo`.  Instead, please make this inherit `ITraitInfoInterface` and have the ModuleInfos implement `ITraitInfo, ITraitInfoInterface`.